freedesktop.org.xml.in

```diff
@@ -6316,4 +6320,15 @@ command to generate the output files.
     <glob pattern="*.stl"/>
   </mime-type>
 
+<mime-type type="application/x-gcode">
+    <_comment>G-code file</_comment>
+    <sub-class-of type="text/plain"/>
+    <generic-icon name="text-x-generic"/>
+    <magic priority="10">
+      <match type="string" value="; " offset="0"/>
+      <match type="string" value="( " offset="0"/>
```

Comment:
The parenthesis are just another way of commenting G-code. As in http://www.cerebralmeltdown.com/download/CamBamG-codeDXFandSTL/gcode/smile.nc for instance.